src/parenthesis.ts

```diff
@@ -67,19 +63,22 @@ export class Parenthesis extends Modifier {
     super();
 
     this.position = position ?? Modifier.Position.LEFT;
+    if (this.position == Modifier.Position.RIGHT) {
+      this.text = '\uE0F6'; /*noteheadParenthesisRight*/
+    } else if (this.position == Modifier.Position.LEFT) {
+      this.text = '\uE0F5'; /*noteheadParenthesisLeft*/
+    }
 
-    this.point = Tables.currentMusicFont().lookupMetric('parenthesis.default.point') ?? Note.getPoint('default');
-    this.setWidth(Tables.currentMusicFont().lookupMetric('parenthesis.default.width'));
+    this.setFontSize(Note.getPoint('default'));
```

Comment:
This can be a different PR, but I think Note.getPoint() could have a clearer method name. Does it relate to font size, or to a point in 2D space?

Reply:
No longer used.